anda/misc/tracy/tracy.spec

```diff
@@ -0,0 +1,65 @@
+Name:			tracy
+Version:		0.10
+Release:		1%?dist
+Summary:		A real time, nanosecond resolution, remote telemetry, hybrid frame and sampling profiler for games and other applications.
+License:		BSD-3-Clause
+URL:			https://github.com/wolfpld/tracy
+Source0:		https://github.com/wolfpld/tracy/archive/refs/tags/v%version.tar.gz
+BuildRequires:  cmake meson gcc libxkbcommon dbus-devel libglvnd glfw-devel freetype-devel pkgconfig(capstone) pkgconfig(libunwind) pkgconfig(libdebuginfod) pkgconfig(tbb)
+Patch:          https://github.com/wolfpld/tracy/commit/1a971d867d6fa5bf6dc57d705dcbbc6020031e7a.patch
```

Comment:
Patch failed in CI

Reply:
I have no idea why this is, can you double check formatting? It works on my local machine just fine.